src/tests/audio/tasks/speech_enhancement_test.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for the speech enhancement task."""
+import os
+
+import pytest
+from speechbrain.inference.separation import SepformerSeparation as separator
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.preprocessing.preprocessing import resample_audios
+from senselab.audio.tasks.speech_enhancement.api import enhance_audios
+from senselab.audio.tasks.speech_enhancement.speechbrain import SpeechBrainEnhancer
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import HFModel
+
+
+@pytest.fixture
+def sample_audio() -> Audio:
+    """Fixture for sample audio."""
+    mono_audio = Audio.from_filepath(("src/tests/data_for_testing/audio_48khz_mono_16bits.wav"))
+    resampled_audios = resample_audios([mono_audio], 16000)
+    return resampled_audios[0]
+
+@pytest.fixture
+def stereo_audio() -> Audio:
+    """Fixture for sample stereo audio."""
+    stereo_audio = Audio.from_filepath("src/tests/data_for_testing/audio_48khz_stereo_16bits.wav")
+    resampled_audios = resample_audios([stereo_audio], 16000)
+    return resampled_audios[0]
+
+@pytest.mark.skipif(os.getenv("GITHUB_ACTIONS") == "true", reason="Don't want to hit the GitHub API limit")
+@pytest.fixture
+def hf_model() -> HFModel:
+    """Fixture for Hugging Face model."""
+    return HFModel(path_or_uri="speechbrain/sepformer-wham16k-enhancement")
+
+@pytest.mark.skipif(os.getenv("GITHUB_ACTIONS") == "true", reason="Don't want to hit the GitHub API limit")
+def test_enhance_audios_stereo_audio(stereo_audio: Audio, hf_model: HFModel) -> None:
+    """Test that enhancing stereo audios raises a ValueError."""
+    with pytest.raises(ValueError, match="Audio waveform must be mono"):
+        SpeechBrainEnhancer.enhance_audios_with_speechbrain(audios=[stereo_audio], model=hf_model)
+
+@pytest.mark.skipif(os.getenv("GITHUB_ACTIONS") == "true", reason="Don't want to hit the GitHub API limit")
+def test_enhance_audios(sample_audio: Audio, hf_model: HFModel) -> None:
```

Comment:
So this is also something I have done within my own code and I question whether these tests are good practices. Should we have any expectations on what the results of enhanced_audios should be? Right now we're testing it at a very basic level that doesn't really guarantee functionality but instead guarantees runability and compilation.

Reply:
You are so right. For now, we are just testing the basics. Later on we will work on making senselab robuster.